src/utils/experimentUpdatesHandler.js

```diff
@@ -68,12 +70,18 @@ const onGEM2SUpdate = (update, dispatch, experimentId) => {
 };
 
 const onWorkResponseUpdate = (update, dispatch, experimentId) => {
-  const { request, response: { error } } = update;
+  const { request: { body }, response: { error } } = update;
 
-  if (request.body.name === 'ClusterCells') {
-    if (error) throw new Error(error);
+  if (error) throw new Error(error);
 
+  if (body.name === 'ClusterCells') {
     dispatch(updateCellSetsClustering(experimentId));
+    pushNotificationMessage('success', endUserMessages.SUCCESS_CELL_SETS_RECLUSTERED);
+  }
+
+  if (body.name === 'GetExpressionCellSets') {
+    dispatch(loadCellSets(experimentId, true));
+    pushNotificationMessage('success', endUserMessages.SUCCESS_NEW_CLUSTER_CREATED);
   }
```

Comment:
I thought that the idea is to call `loadCellSets` from within the individual action for creating cell sets based on gene expression, if the worker-results bucket contains results